dawn.json

```diff
@@ -1028,15 +1036,23 @@
                 "args": [
                     {"name": "num mip levels", "type": "uint32_t"}
                 ]
-            },
-            {
-                "name": "set allowed usage",
-                "args": [
-                    {"name": "usage", "type": "texture usage bit"}
-                ]
             }
         ]
     },
+    "texture descriptor": {
+        "category": "structure",
+        "extensible": true,
+        "members": [
+            {"name": "usage", "type": "texture usage bit"},
+            {"name": "dimension", "type": "texture dimension"},
+            {"name": "width", "type": "uint32_t"},
```

Comment:
FYI: at some point `width`, `height` and `depth` will probably become a nested `dawn::Extent3D` structure like in WebGPU. We can do it when the WebGPU API crystallizes a bit more though.